src/edu/csus/ecs/pc2/clics/API202003/EventFeedJSON.java

```diff
@@ -464,13 +456,16 @@ public String getClarificationJSON(IInternalContest contest) {
         return stringBuilder.toString();
     }
 
-    String getClarificationJSON(IInternalContest contest, Clarification clarification, ClarificationAnswer clarAnswer) {
+    public String getClarificationJSON(IInternalContest contest, Clarification clarification, ClarificationAnswer clarAnswer) {
         return jsonTool.convertToJSON(clarification, clarAnswer).toString();
 
     }
 
     public String getAwardJSON(IInternalContest contest) {
-        return awardJSON.createJSON(contest);
+        return null;
+        // TODO: Old code return null as a result of the unimplement createJSON in the unimplemented AwardJSON class (which has
+        // since been removed.  This should use the IJSONTool interface which is API specific
+        //return awardJSON.createJSON(contest);
```

Comment:
Same comment as above:  should this throw an exception?   (I can see us getting to a WF and suddenly finding that we cannot generate Awards because this got lost in the myriad of "TODOs" that we never got A Round Tuit for....)

Reply:
The old award code did nothing but return null.  Since we are not supporting new features in 202003 anymore, there was no point in keeping the awards class around.  The 202306 API DOES support awards correctly (I'd like to think).

Reply:
ok.